test/unit/helpers/xhr/MFA_CHALLENGE_webauthn.js

```diff
@@ -0,0 +1,47 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2018-08-08T14:36:45.000Z",
+    "status": "MFA_CHALLENGE",
+    "_embedded": {
+      "user": {
+        "id": "00uk9z38LFBdMXeWw0g3",
+        "passwordChanged": "2018-07-28T12:56:58.000Z",
+        "profile": {
+          "login": "xxx@xxx.xxx",
+          "firstName": "xxx",
+          "lastName": "xxx",
+          "locale": "en_US",
+          "timeZone": "America/Los_Angeles"
+        }
+      },
+      "factor": {
+        "id": "webauthnFactorId",
+        "factorType": "webauthn",
+        "provider": "FIDO",
+        "vendorName": "FIDO",
+        "profile": {"credentialId": "vdCxImCygaKmXS3S_2WwgqF1LLZ4i_2MKYfAbrNByJOOmSyRD_STj6VfhLQsLdLrIdgvdP5EmO1n9Tuw5BawZw"},
+        "_embedded": {
+          "challenge": {
+            "challenge": "kygOUtSWURMv_t_Gj71Y",
```

Comment:
Probably this was discussed in the api review, but it seems a little redundant to have `challenge.challenge`... it used to be `challenge.nonce`.

Reply:
Yeah, the upper level `challenge` is to be consistent with other factor API structure, the nested `challenge` is to be consistent with webauthn protocol (https://www.w3.org/TR/webauthn/#dictdef-publickeycredentialrequestoptions).